### DISCUSSION TITLE
I would like an option to replace the menu bar icon with a croissant

### DISCUSSION THREAD

**Discussion by smammy** [rxhanson/Multitouch-Community#176] [Ideas]
The other day I noticed that Multitouch’s menu bar icon kinda looks like a croissant. I am now obsessed with it. Please add an option to replace the current icon with a croissant.

| Ol’ Busted | New Hotness |
| --- | --- |
| <img width="24" alt="image" src="https://github.com/rxhanson/Multitouch-Community/assets/2546386/ba307b96-ffd6-482d-87ef-798115807b56"> | <img width="24" alt="image" src="https://github.com/rxhanson/Multitouch-Community/assets/2546386/64c56ea2-27b0-44b7-828f-2f818300a3be"> |

Thank you for considering this important issue.

**Comment by rxhanson** [comment]
Ha! Love it. Now that you mention it, I might not be able to unsee it. Oddly enough, I did have an idea where I wanted to let the user pick from a selection of different animals to use as the menu bar icon. If I follow through with that, a croissant will absolutely be added.

**Reply by smammy** [reply]
Sounds awesome. Thanks!
